code/defines/procs/announce.dm

```diff
@@ -0,0 +1,121 @@
+/var/datum/announcement/priority/priority_announcement = new(do_log = 0)
+/var/datum/announcement/priority/command/command_announcement = new(do_log = 0, do_newscast = 1)
+
+/datum/announcement
+	var/title = "Attention"
+	var/announcer = ""
+	var/log = 0
+	var/sound
+	var/newscast = 0
+	var/channel_name = "Station Announcements"
+	var/announcement_type = "Announcement"
+	var/disable_newscasts = 1 // Bay also adds announcements to their newscaster system - set this to 0 to also use that system
```

Comment:
That's a good point, although I did intentionally choose for this set-up. We might decide to use the newscasters at a later point, so in case that happens I wanted to make it a very simple switch to re-enable them (just removing the disable_newscasts variable completely), without having to modify several files to ensure the newscaster setting is set to 0. A bit lazy, I know, but this variable shouldn't be in too long.